client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,48 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakup.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakup = ( event ) => {
+	if ( ! event?.fee_rates?.history ) {
+		return [];
+	}
+
+	const {
+		fee_rates: { history },
+	} = event;
+
+	const feeLabelMapping = {
+		base: __( '\u00A0 • Base fee %s%%', 'woocommerce-payments' ),
+		'additional-international': __(
+			'\u00A0 • International fee %s%%',
+			'woocommerce-payments'
+		),
+		'additional-fx': __(
+			'\u00A0 • Foreign exchange fee %s%%',
+			'woocommerce-payments'
```

Comment:
I would exclude these characters `\u00A0 •` from the i18n function as they don't need translations.